sql/load_demo_data.sql

```diff
@@ -3106,6 +3232,10 @@ CREATE TABLE `oh_medicaldsrstockmovward` (
   KEY `FK_MEDICALDSRSTOCKMOVWARD_PATIENT_idx` (`MMVN_PAT_ID`),
```

Comment:
In this table the column MMVN_WRD_ID_A has the default value in the demo data loading as NULL where the multiple steps sets it to '' (empty string).
The multiple step version for this table has:
<pre>
  `MMVN_WRD_ID_A` char(3) NOT NULL DEFAULT '',
</pre>
where as the load demo data has:
<pre>
  `MMVN_WRD_ID_A` char(3) NOT NULL,
</pre>